src/app/components/profile/profile.component.ts

```diff
@@ -47,7 +51,7 @@ export class ProfileComponent implements OnInit {
    * Modal display flag
    */
   isTokenModalVisible = false;
-
+  
```

Comment:
Please revert this change.

Reply:
but sir there are deleted lines you said me to revert

Reply:
yeah I mean to undo that change, make it as it is in the master branch of main repo. Please remove unnecessary change which we don't need in this PR.

Reply:
done sir